src/main/scala/mesosphere/marathon/core/instance/update/InstanceUpdateOpResolver.scala

```diff
@@ -31,12 +31,24 @@ private[marathon] class InstanceUpdateOpResolver(
     */
   def resolve(op: InstanceUpdateOperation)(implicit ec: ExecutionContext): Future[InstanceUpdateEffect] = {
     op match {
+      case op: Schedule =>
+        createInstance(op.instanceId) {
+          // TODO(karsten): Create events
```

Comment:
Is it going to be done in this PR?

Reply:
I'm afraid the diff would be too big. I added the JIRA to the epic https://jira.mesosphere.com/browse/MARATHON-8225.